### ISSUE TITLE
[Usable – Make Code – Home page] Narrator focus is landing on decorative image when navigating in Scan mode under ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3546]
**User Experience**:
User who depends upon screen reader will be confused if screen reader just mention image and may think that the information inside the image is missing for AT dependent users.

**Note**: User credentials should NOT be included in the bug. 

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Observe the narrator behavior in scan mode as focus moves on decorative image or not.

**Actual Result:**
Narrator focus is landing on decorative image when navigating in Scan mode under ‘Home page’ and does not provide any information.

**Expected Result:**
Whenever the user navigating with the narrator in scan mode then the narrator focus does not move to decorative image as Alt =" " is defined for the image available on the ‘Home page’ page.

**Note:**
This issue does not exists on Firefox & Chrome.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)
Screen Reader: Narrator

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev4, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #Win10-Edge(Chromium)

[Usable_Narrator focus moves on decorative image.zip](https://github.com/microsoft/pxt-microbit/files/5473761/Usable_Narrator.focus.moves.on.decorative.image.zip)

**Comment by jwunderl** [comment]
No repro on this one that I see, but I did notice along the way that the cycling top carousel needs titles / aria labels for the jump buttons in the bottom right 
![image](https://user-images.githubusercontent.com/5615930/113216295-f74c7280-9230-11eb-891e-c616d43700b3.png)

which is similar to this issue, as they just read out 'button' right now.
